app/config/services/services.xml

```diff
@@ -174,5 +174,8 @@
 
         <!-- History -->
         <service id="AppBundle\History\EmailSubscriptionHistoryHandler" />
+
+        <!-- Stats -->
+        <service id="AppBundle\Statistics\StatisticsParametersFilter" public="true" />
```

Comment:
This class should not be a service as it is something mutable. 

It's just a place where we store all filters asked by the client.

It could be instantiated like this in controller instead `StatisticsParametersFilter::fromRequest()`